src/inputs/checkbox-radio.scss

```diff
@@ -30,31 +29,42 @@
 
   .iui-checkbox-checkmark,
   .iui-radio-dot {
-    box-sizing: border-box;
     width: $iui-icons-default;
     height: $iui-icons-default;
     margin: 0;
     display: flex;
+    flex-shrink: 0;
     justify-content: center;
     align-items: center;
-
-    @include themed {
-      border: 1px solid rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
-      background-color: t(iui-color-background-1);
+    position: relative;
+    isolation: isolate;
+
+    &::after {
+      content: '';
+      position: absolute;
+      inset: 0;
+      z-index: -1;
+      @include themed {
+        border: 1px solid rgba(t(iui-color-foreground-body-rgb), t(iui-opacity-4));
+        background-color: t(iui-color-background-1);
+      }
+      @media (prefers-reduced-motion: no-preference) {
+        transition: border-color $iui-speed-fast ease-out;
+      }
     }
 
     svg:not(.iui-radial) {
-      width: $iui-sm;
-      height: $iui-sm;
+      width: 100%;
+      height: 100%;
```

Comment:
Maybe it makes sense to keep the explicit size?

Reply:
The only benefit I can see for going with 100% is if the user changes the size of the checkbox / radio button, the interior will scale accordingly.  

With 100%:
![Screen Shot 2022-01-07 at 1 55 44 PM](https://user-images.githubusercontent.com/849817/148592957-435658dd-ff9a-426c-8245-c448145e528f.png)

With explicit size:
![Screen Shot 2022-01-07 at 1 57 35 PM](https://user-images.githubusercontent.com/849817/148593171-ff796990-eb63-40b3-a408-963cd4aad544.png)